FirstProject/css/feedback.css

```diff
@@ -89,15 +78,16 @@ h1.interview_text_area {
     }
 
     .left_image {
-        float: left!important;
+        width: 100%;
     }
 
     header form {
         margin-left: 0!important;
     }
 
     .header_form_section {
-        width: 94%!important;
+        width: 87%;
+        margin: 0 40px auto auto;
```

Comment:
Not at all. I just found it a bit odd that you had a `margin-bottom` of `auto` (top and bottom `auto` margins are [treated as 0](https://stackoverflow.com/questions/12415661/using-marginauto-to-vertically-align-a-div), by default), so I wanted to make sure that you had the right order. That seems to be the case, so I'm satisfied with the answer, irrespective of what the resulting layout would look like (if it looks OK to you, it's enough for me :wink:).